packages/react-ui/src/widgets/legend/LegendWrapper.js

```diff
@@ -82,61 +82,65 @@ export default function LegendWrapper({
       />
       {hasChildren && !!children && (
         <Collapse ref={wrapper} in={expanded} timeout='auto' unmountOnExit>
-          <Box className={classes.content}>
+          <Box sx={{ pt: 0, px: 2, pb: 3 }}>
```

Comment:
Even if the sx values seem equal to original, the UI renders differently.

I was still getting partially 'hidden' copy, until I changed a bit the values.

Left: previous / Center: this branch / Right: the modified values 
![image](https://user-images.githubusercontent.com/458196/235227828-78e6afdf-f7d0-4e2e-bf24-7ad25eb3d353.png)

The problem to me is that the overall structure is here quite messy. After moving away from makeStyles I think we should make a deep refactor and cleanup of the component

Reply:
@vmilan I'm gonna move on with this approach, to land PR and unblock the rest, but we should check it in the mid-term